Sources/Swift/Integrations/SessionReplay/RRWeb/SentryRRWebOptionsEvent.swift

```diff
@@ -0,0 +1,18 @@
+@_implementationOnly import _SentryPrivate
+import Foundation
+
+@objc class SentryRRWebOptionsEvent: SentryRRWebCustomEvent {
```

Comment:
Calling these `...Event` is a big confusing. It looks like it's being appended to an array of breadcrumbs, can we do some renaming for this?

Reply:
We're following RRWeb naming. This is called an event for RRWeb. for android we're doing the same.